grammar.js

```diff
@@ -26,7 +26,7 @@ module.exports = grammar({
   extras: $ => [
     $.comment,
     $._line_break,
-    /[ \t\r]/
+    /[ \t\r\uFEFF]/
```

Comment:
This will probably come up in other languages. We should fix tree-sitter's implementation of the `\s` character class to include non-breaking spaces, since JavaScript's regexps seem to do this.